scu-de/src/fiskaltrust.Middleware.SCU.DE.Swissbit/SwissbitSCU.cs

```diff
@@ -738,7 +742,20 @@ public async Task<StartExportSessionResponse> StartExportSessionAsync(StartExpor
 
                 await UpdateTimeAsync(GetProxy());
                 SetExportState(exportId, ExportState.Running);
-                CacheExportAsync(exportId, request.ClientId, request.Erase).FireAndForget();
+
+                if (_configuration.ChunkExportTransactionCount > 0)
+                {
+                    if (_lastTransactionNr == 0)
+                    {
+                        _logger.LogWarning("Before executing a partial export a daily closing has to be made.");
+                        throw new Exception("Missing Daily Closing.");
```

Comment:
I think we should log an error here not warning if we crash. or if we just throw maybe that will that be logged later? I'm not sure 😅